Lib/times.py

```diff
@@ -135,12 +135,11 @@ def isMonthly(s):
     units=tim.units
     monthly=1
     for i in range(len(tim)-1):
-      month1=cdtime.reltime(tim[i],units).torel('months since 2000').value
-      month2=cdtime.reltime(tim[i+1],units).torel('months since 2000').value
+      month1=cdtime.reltime(tim[i],units).torel('months since 2000',tim.getCalendar()).value
```

Comment:
@doutriaux1 glad to see that you have fallen into line and are now importing using 3-letter acronyms